rg_instructor_analytics/static/rg_instructor_analytics/js/GradebookTab.js

```diff
@@ -76,22 +76,29 @@ function GradebookTab(button, content) {
 
 
         $(greadebookTab.gradebookTableBody).click(function (element) {
+            let colorArray = greadebookTab.examNames.map((item, idx, arr)=>{
+                if (idx === arr.length - 1){
+                    return '#c14f84';
+                }
+                return '#568ecc';
+            })
+
             var stat = {
                 y: greadebookTab.studentInfo[element.target.dataset['position']].grades,
                 x: greadebookTab.examNames,
                 type: 'bar',
                 marker:{
-                    color: ['#568ecc', '#568ecc','#568ecc','#568ecc','#568ecc','#c14f84']
+                    color: colorArray
                 },
-                width: 0.6,
+                width: 0.6
```

Comment:
Why do you remove the comma, it as usual practice to have a comma at the end of some container content ends?